### DISCUSSION TITLE
Possibility to delete images

### DISCUSSION THREAD

**Discussion by Ninos** [voidlabs/mosaico#683] [Ideas]
Currently it seems there's no possibility to delete uploaded images via user interface. As tested before weeks it's also not possible to use images with spacing/special chars.

**Comment by bago** [comment]
Mosaico doesn't offer a delete image feature.

We are not aware of issues with spacing/special chars: open a bug with complete informations about this.

Please read this before opening more issues:
https://github.com/voidlabs/mosaico/blob/master/CONTRIBUTING.md

And please use Discussion, first, if you are not sure you are reporting a bug:
https://github.com/voidlabs/mosaico/discussions

**Comment by Ninos** [comment]
Thx for your reply @bago 

I created an issue because I thought it's the better place for a feature request of such basic thing, which don't really need any discussion.. :-)

If it's a won't-fix, just close this request, I'll then need to live with that :D
